src/language-html/utils.js

```diff
@@ -494,6 +494,18 @@ function getNodeCssStyleDisplay(node, options) {
     case "ignore":
       return "block";
     default:
+      if (
+        node.type === "element" &&
+        !node.hasExplicitNamespace &&
+        node.namespace &&
+        !["html", "svg"].includes(node.namespace)
+      ) {
+        const display = CSS_DISPLAY_TAGS[node.name];
+        if (display) {
+          return display;
+        }
+      }
+
       return (
         (node.type === "element" &&
           (!node.namespace || isInSvgForeignObject) &&
```

Comment:
Can't we just add `|| !["html", "svg"].includes(node.namespace)` to this `if` instead?